Common/src/main/java/com/tabnineCommon/lifecycle/WorkspaceListenerService.kt

```diff
@@ -57,6 +58,14 @@ class WorkspaceListenerService {
                 Logger.getInstance(javaClass).debug("$url in project ${project.name} has unsupported protocol (${url.protocol})")
                 continue
             }
+            if (rootPaths.any { it.startsWith(url.path) }) {
```

Comment:
If the rootPath appears before its parent it would still be included. Better check this after you have the full list